drivers/wireless/ieee80211/bcm43xxx/bcmf_interface.h

```diff
@@ -140,15 +117,15 @@ void bcmf_initialize_interface_frames(void);
  ****************************************************************************/
 
 void bcmf_interface_free_frame(FAR struct bcmf_dev_s  *priv,
-                               bcmf_interface_frame_t *iframe);
+                               FAR bcmf_interface_frame_t *iframe);
 
 /****************************************************************************
  * Name: bcmf_interface_allocate_frame
  ****************************************************************************/
 
-bcmf_interface_frame_t
+FAR bcmf_interface_frame_t
 *bcmf_interface_allocate_frame(FAR struct bcmf_dev_s *priv,
```

Comment:
I think it is better to move `*` to row above

Reply:
Done